### ISSUE TITLE
HIDA scan: bullseye image

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10050]
**Preferred term label:**
HIDA scan: bullseye image

**Synonyms**
hepatobiliary iminodiacetic acid scan

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0025155

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Gallbladder volvulus

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
I am going to suggest we do not make a term for this, since outside of one 1987 publication that I have no access to, I cannot find a single publication that describes this (I found two that refer to the 1987 publication). Therefore, I think that it is unlikely that this terminology is actually used by current practitioners.
Please reopen if my assessment is incorrect.

**Comment by MickeySegal** [comment]
We got it from https://www.ncbi.nlm.nih.gov/books/NBK448064/ that purports to be discussing current practice:
"A HIDA scan, if performed, shows a bullseye image due to the subsequent accumulation of the radio isotope in the gallbladder."
